src/client/mod.rs

```diff
@@ -17,7 +17,10 @@ use token::{Token, Lifetime, Refresh};
 
 /// OAuth 2.0 client.
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct Client<P: Provider> {
+pub struct Client<P> {
```

Comment:
I seemed to have discovered a rabbit hole of semantic debate bordering on tabs vs spaces on the users discourse about this! I'll have to go and try to make sure I'm being consistent in my crates now too.